superset/assets/javascripts/SqlLab/components/VisualizeModal.jsx

```diff
@@ -133,12 +133,13 @@ class VisualizeModal extends React.PureComponent {
   }
   buildVisualizeAdvise() {
     let advise;
+    const timeout = this.props.timeout;
     const queryDuration = moment.duration(this.props.query.endDttm - this.props.query.startDttm);
-    if (Math.round(queryDuration.asMilliseconds()) > QUERY_TIMEOUT_THRESHOLD) {
```

Comment:
Can anyone tell which is the new constant supplanting QUERY_TIMEOUT_THRESHOLD.

Reply:
SUPERSET_WEBSERVER_TIMEOUT

Reply:
Thanks @xrmx . why this name got changed ?